src/service/data_products/taxa_count.py

```diff
@@ -344,9 +348,22 @@ async def _add_subset_data_in_place(
         )
         name, count = names.FLD_TAXA_COUNT_NAME, names.FLD_TAXA_COUNT_COUNT
         mqd = {d[name]: d[count] for d in matchq}
+
+        missing_names = set(mqd.keys()) - set(d[name] for d in q)
```

Comment:
There's a potential optimization here to skip this step if the last sort is `standard` since any missing names will be cut off of the end of the list, but I think we can worry about that when it becomes an issue

Reply:
👍 added a todo item. Given the size of taxa count I think it should be okay for now.